Documentation/RC1SupportedScenarios.md

```diff
@@ -0,0 +1,24 @@
+WCF in RC1
+===========================
+
+WCF Features
+------------
+
+
+![Image](http://findicons.com/files/icons/573/must_have/16/check.png) -- Supported: Release quality
+
+![Image](http://findicons.com/files/icons/1155/diagona/16/104.png) --  Supported: Not fully tested
+
+![Image](http://findicons.com/files/icons/573/must_have/16/add.png) -- Not Yet Supported: Future release
+
+![Image](http://findicons.com/files/icons/573/must_have/16/delete.png) -- Not Supported: Will not support
+
```

Comment:
Hmmm -- I think a "will not support" scenario belongs in a different table.  Otherwise we will always have tables with red X's and make the product look incomplete.  And I think something like the red X is the right icon for "Not yet supported".  Ideally the table will contain only "supported", "not yet tested", and "not yet supported".

Out of curiosity, it is possible to make the "not yet supported" items also contain a link to an issue?  Or does that blow out the size of the columns?